pilot/pkg/bootstrap/server.go

```diff
@@ -152,6 +152,7 @@ type PilotArgs struct {
 	MeshConfig       *meshconfig.MeshConfig
 	CtrlZOptions     *ctrlz.Options
 	Plugins          []string
+	Debug            bool
```

Comment:
We generally access the fields for features/pilot directly, no need to change the bootstrap.